bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -59,7 +58,6 @@ trait ItcTable extends Table {
   peer.setRowSelectionAllowed(true)
   peer.setColumnSelectionAllowed(false)
   peer.setSelectionMode(ListSelectionModel.SINGLE_SELECTION)
-  peer.getTableHeader.setReorderingAllowed(false)
```

Comment:
There's no reason for it, I guess, apart from the fact that the default ordering makes sense.
I was just maintaining what was already in place.